app/models/speaker/profiles.rb

```diff
@@ -0,0 +1,27 @@
+class Speaker::Profiles < ActiveRecord::AssociatedObject
+  performs do
+    retry_on StandardError, attempts: 3, wait: :polynomially_longer # TODO: replace with `retries: 3`.
+    limits_concurrency key: -> { _1.id }
+  end
+
+  def enhance_all_later
+    enhance_with_github_later
+    enhance_with_bsky_later
+  end
```

Comment:
I know not everybody likes callbacks, but I'm showing what we could also consider doing with ActiveRecord::AssociatedObject:

```suggestion
  # has_object :profiles, after_save: true
  def after_save
    enhance_with_github_later if speaker.github_previously_changed?
    enhance_with_bsky_later   if speaker.bsky_previously_changed?
  end
```